universal-builds/universal-builds.md

```diff
@@ -72,15 +72,37 @@ This method is useful for manual deployments.
 In the kibana.yml configuration file, you have the option to specify the mode for retrieving activation keys. 
 Setting this parameter effectively means that all other activation key retrieval methods will be disregarded.
 
-#### For environment variable option, you can add to `kibana.yml`:
+#### Default behaviour
+
+If there is no `kibana.yml` license config provided or 
+
+```yaml
+readonlyrest_kbn:
+    license:
+        activationKeyRetrievalMode: "all" # "file" | "env" | "all" | "none"
+```
+
+defined, the order of AK checking the most recent AK is:
```

Comment:
"the most recent" ? Does it mean it checks in multiple places and elects the newest? Strange because later you say that we just stop evaluating when an AK is found?

Reply:
Right, it's misleading. What I mean here, is that the order of activation key validation. I rephrased it